packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/bin/gen_l10n.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Merge ARB files from sources and generate localizations
+echo "Merging ARB files..."
+dart run bin/merge_arb_files.dart
+
+echo ""
+echo "Generating localizations..."
+dart run intl_utils:generate
```

Comment:
Does this line expect the user to have `intl_utils` installed locally? If so, this should be documented.